data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -196,3 +197,54 @@
 HOLC_GRADE_D_TRACT_20_PERCENT_FIELD: str = "Tract is >20% HOLC Grade D"
 HOLC_GRADE_D_TRACT_50_PERCENT_FIELD: str = "Tract is >50% HOLC Grade D"
 HOLC_GRADE_D_TRACT_75_PERCENT_FIELD: str = "Tract is >75% HOLC Grade D"
+
+
+# Climate Change
+EXPECTED_POPULATION_LOSS_RATE_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for expected population loss rate and is low income"
+EXPECTED_AGRICULTURE_LOSS_RATE_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for expected agriculture loss rate and is low income"
+EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for expected building loss rate and is low income"
+
+# Clean energy and efficiency
+PM25_LOW_INCOME = "PM2.5 exposure and is low income"
+ABOVE_90TH_FOR_ENERGY_BURDEN_LOW_INCOME = f"At or above the {PERCENTILE}th percentile for energy burden and is low income"
+# Clean transportation
+DIESEL_PARTICULATE_MATTER_LOW_INCOME = f"At or above the {PERCENTILE}th percentile for diesel particulate matter and is low income"
+TRAFFIC_PROXIMITY_MATTER_LOW_INCOME = "Traffic Proximity and is low income"
+# Affordable and Sustainable Housing
+LEAD_PAINT_MEDIAN_HOME_VALUE = f"At or above the {PERCENTILE}th percentile for lead paint and the median house value is less than 90th percentile"
+HOUSING_BURDEN_LOW_INCOME = f"At or above the {PERCENTILE}th percentile for housing burden and is low income"
+# Remediation and Reduction of Legacy Pollution
+RMP_LOW_INCOME = f"At or above the {PERCENTILE}th percentile for proximity to RMP sites and is low income"
+SUPERFUND_LOW_INCOME = f"At or above {PERCENTILE}th percentile for proximity to superfund sites and is low income"
+HAZARDOUS_WASTE_LOW_INCOME = f"At or above above {PERCENTILE}th percentile for proximity to hazardous waste facilities and is low income"
+# Critical Clean Water and Waste Infrastructure
+WASTEWATER_DISCHARGE_LOW_INCOME = f"At or above above {PERCENTILE}th percentile for wastewater discharge and is low income"
+# Health Burden
+DIABETES_LOW_INCOME = (
+    f"At or above the {PERCENTILE}th percentile for diabetes and is low income"
+)
+ASTHMA_LOW_INCOME = (
+    f"At or above the {PERCENTILE}th percentile for asthma and is low income"
+)
+HEART_DISEASE_LOW_INCOME = f"At or above the {PERCENTILE}th percentile for heart disease and is low income"
+LIFE_EXPECTANCY_INCOME = f"At or above the {PERCENTILE}th percentile for life expectancy and is low income"
+
+ACHIEVEMENT_TEMPLATE_FOR_WORKFORCE = """and where the high school degree achievement
```

Comment:
i really think "and where the high school degree achievement rate for adults 25 years and older is less than {PERCENTILE}" should be replaced by "and low HS education". you can remove the specific HS education percentile

Reply:
Okay so I have now just, for example, `At or above the {PERCENTILE}th percentile for households
    and low HS education`

Reply:
Well, it depends on what the variable is. In your example, you just say "percentile for households" but it doesn't reference what field those households is exceeding. 

If you're setting `UNEMPLOYMENT_LOW_HS_EDUCATION`, for instance, you'll want `At or above the {PERCENTILE}th percentile for unemployment and low HS education`.

Reply:
Ah - thanks for pointing that out - `POVERTY_LOW_HS_EDUCATION_FIELD` is the field and I am just following what was in the design document:

`% households at or below 100% federal poverty level` and `Where the high school degree achievement rate for adults 25 years and older is less than 90%`